src/engine/engine.ts

```diff
@@ -783,7 +784,7 @@ function autosellJunk(): void {
   // Sell extra consumables (after 1 has been absorbed)
   for (const item_name in getInventory()) {
     const item = Item.get(item_name);
-    if (consumables_blacklist.has(item)) continue;
+    if (consumables_blacklist.has(item) || historicalPrice(item) > 5000 || !item.tradeable || item.quest || item.gift) continue;
```

Comment:
I think the `historicalPrice > 5000` will also stop us from autoselling meat pulls. Maybe `(historicalPrice > 5000 && autosellPrice < 5000)` or something, so we still autosell the high-meat things?

Reply:
Done!